pyproject.toml

```
@@ -1,8 +1,12 @@
 [tool.poetry]
 name = "lnbits"
-version = "0.1.0"
-description = ""
-authors = ["matthewcroughan <matt@croughan.sh>"]
+version = "0.9.5.3"
+description = "LNbits, free and open-source Lightning wallet and accounts system."
+authors = [
+    "benarc <ben@lnbits.com>",
+    "dni <dni@lnbits.com>",
+    "matthewcroughan <matt@croughan.sh>"
+]
```

```suggestion
authors = [
    "lnbits <lnbits@lnbits.com>",
]
```

Does this address exist @arcbtc?

---

08c485e